### ISSUE TITLE
Analysing multi-lane 10X Chromium data

### ISSUE THREAD

**Issue by johnchamberlin** [sdparekh/zUMIs#127]
Hello,

I'm trying to re-analyze some sample data from the 10X Genomics website. However, I'm finding that the reads are generally split across multiple files. For example, here are the fastqs for the following small dataset: https://support.10xgenomics.com/single-cell-gene-expression/datasets/3.0.0/hgmm_1k_v3

hgmm_1k_v3_fastqs/hgmm_1k_v3_S1_L001_R2_001.fastq.gz
hgmm_1k_v3_fastqs/hgmm_1k_v3_S1_L002_I1_001.fastq.gz
hgmm_1k_v3_fastqs/hgmm_1k_v3_S1_L001_R1_001.fastq.gz
hgmm_1k_v3_fastqs/hgmm_1k_v3_S1_L002_R1_001.fastq.gz
hgmm_1k_v3_fastqs/hgmm_1k_v3_S1_L002_R2_001.fastq.gz
hgmm_1k_v3_fastqs/hgmm_1k_v3_S1_L001_I1_001.fastq.gz

As you can can see, there are two barcode files (R1) and two cDNA files(R2), and two index files (can I ignore these?)

How do I specify this data format for zUMIs? Should I first concatenate them?

Cheers,
John

**Comment by cziegenhain** [comment]
Hi John,

Thanks for using zUMIs!
Yes you are right, you would concatenate the fastq files before setting up the run.
If it's all one sample, you also don't need the index read that has the i7 barcode. 
Let me know if I can help you further.

Best,
Christoph

**Comment by johnchamberlin** [comment]
Thanks Christoph. I concatenated the files and it seem to be working fine. 

By the way, since this is a mixed species dataset, I used CellRanger to generate the combined mouse & human reference fasta and GTF (cellranger mkref: https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/advanced/references#multi) - this was convenient because it automatically renames the chromosomes in the resulting fasta and GTF. Then I used your provided code to construct the STAR index from the combined fasta without the GTF.
